### DISCUSSION TITLE
14/5/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#148] [Daily Sprint]
Que hice ayer
Que hago hoy
Impedimentos

**Comment by Pamela198713** [comment]
Ayer lunes revise documentación para ver qué faltaba. Decidimos en equipo la fecha y hs para la revisión con los profes.
Hoy veré qué hace falta, estoy bastante ajustada con los tiempos del colegio y los informes.
El tiempo sigue siendo impedimento y la falta de reorganizar mis actividades.

**Comment by rpgrca** [comment]
Ayer hice algo de documentación para el cierre del sprint. Hoy tengo clases de Emprendedurismo, espero que podamos cerrar la primer entrega. También quiero volver a lo de Android para seguir avanzando con las llamadas a la base de datos para el segundo sprint.

**Comment by fmdcba** [comment]
Ayer estuve con los casos de prueba y el trabajo de emprendedurismo. Hoy voy a estar en la clase y atento a los cambios que se necesiten

**Comment by LuciaL152** [comment]
Ayer estuve haciendo algunos cambios en el prototipo de la app. Hoy lo termine y además estuve revisando un poco el material de estudio de otras materias.
